python/ql/integration-tests/query-suite/python-code-quality.qls.expected

```diff
@@ -1,4 +1,6 @@
 ql/python/ql/src/Functions/NonCls.ql
 ql/python/ql/src/Functions/NonSelf.ql
+ql/python/ql/src/Functions/ReturnConsistentTupleSizes.ql
 ql/python/ql/src/Functions/SignatureSpecialMethods.ql
 ql/python/ql/src/Resources/FileNotAlwaysClosed.ql
+ql/python/ql/src/Variables/LoopVariableCapture/LoopVariableCapture.ql
```

Comment:
The added queries have `precision high`.

Reply:
Yes - those queries needed to be added to the suite anyway.